.github/workflows/slh-dsa.yml

```diff
@@ -0,0 +1,35 @@
+name: slh-dsa
+on:
+  pull_request:
+    paths:
+      - ".github/workflows/slh-dsa.yml"
+      - "slh-dsa/**"
+      - "Cargo.*"
+  push:
+    branches: master
+
+defaults:
+  run:
+    working-directory: slh-dsa
+
+env:
+  CARGO_INCREMENTAL: 0
+  RUSTFLAGS: "-Dwarnings"
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        rust:
+          - 1.75.0 # MSRV
+          - stable
+    steps:
+      - uses: actions/checkout@v4
+      - uses: dtolnay/rust-toolchain@master
+        with:
+          toolchain: ${{ matrix.rust }}
+      - run: cargo check --all-features
+      # - run: cargo test --no-default-features TODO
```

Comment:
Need to address this: tests fail with `--no-default-features`

cc @tjade273

Reply:
addressed in #814